tests/js/atomic-forces.test.ts

```diff
@@ -0,0 +1,120 @@
+import type { AtomicForcesPropertySchema } from "@mat3ra/esse/dist/js/types";
```

Comment:
1. It would be nice to isolate the tests for specific properties to `properties` subdirectory from the rest of the tests codebase
2. The filenames should contain the names of the actual property as we use it - e.g. `atomic_forces`, not `atomic-forces`